cmd/fs_mgmt/src/fs_mgmt.c

```diff
@@ -100,9 +101,60 @@ fs_mgmt_file_download(struct mgmt_ctxt *ctxt)
         }
     }
 
+#ifdef __ZEPHYR__
+    static size_t buff_used = 0;
+    size_t buff_free = MCUMGR_BUF_SIZE -
```

Comment:
I am not a fan of declaring variables in the middle of a function, I would suggest you to move these declarations to the beginning of the function.

Reply:
OK, that could be done, just wanted to keep all zephyr specific stuff within one block.